builds/e2e/templates/longhaul-deploy-windows.yaml

```diff
@@ -34,5 +34,5 @@ steps:
         #Requires -RunAsAdministrator
         $testName="LongHaul"
         $ArtifactInfo=convertfrom-stringdata (get-content $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/artifactInfo.txt -raw)
-        $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/scripts/windows/test/Run-E2ETest.ps1 -E2ETestFolder "$(Agent.HomeDirectory)/.." -ReleaseLabel "${{ parameters['release.label'] }}" -ArtifactImageBuildNumber "$($ArtifactInfo.BuildNumber)" -TestName "$testName" -ContainerRegistry "${{ parameters['container.registry'] }}" -ContainerRegistryUsername "${{ parameters['container.registry.username'] }}" -ContainerRegistryPassword "${{ parameters['container.registry.password'] }}" -IoTHubConnectionString "${{ parameters['iotHub.connectionString'] }}" -EventHubConnectionString "${{ parameters['eventHub.connectionString'] }}" -SnitchBuildNumber "${{ parameters['snitch.build.number'] }}" -SnitchStorageAccount "${{ parameters['snitch.storage.account'] }}" -SnitchStorageMasterKey "${{ parameters['snitch.storage.masterKey'] }}" -SnitchAlertUrl "${{ parameters['snitch.alert.url'] }}" -LoadGenMessageFrequency "${{ parameters['loadGen.message.frequency'] }}"
```

Comment:
missing analyzer.consumerGroupId: '' at line 15.

Reply:
Oops, good catch. I must have checked stress test deploy yaml when I was double-checking. 

Successful deployment here: 
https://msazure.visualstudio.com/One/_build/results?buildId=25895323

Reply:
I SSH'd into each agent running long haul, except for AMD64 which has the wrong key in keyvault. They all report correct consumer group ids in the analyzer logs.